pkg/v1/remote/image.go

```diff
@@ -100,6 +101,11 @@ func (r *remoteImage) RawConfigFile() ([]byte, error) {
 		return nil, err
 	}
 
+	if m.Config.Data != nil && int64(len(m.Config.Data)) == m.Config.Size {
```

Comment:
If len(data) != size, should we fail? That definitely sounds like something is wrong.

The proposed spec says "The decoded data ... SHOULD be verified against the digest and size fields"

Reply:
Yeah I probably should have marked this as a draft, just playing around right now.